go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/jomei/notionapi
+module github.com/yyewolf/notionapi
```

Comment:
Don't think this should be changed as this will go under `github.com/jomei/notionapi`

Reply:
Yep, this was a mistake and shouldn't have been pushed here..